p2p/base/port_allocator.h

```diff
@@ -100,6 +100,8 @@ enum class IceRegatheringReason {
   MAX_VALUE
 };
 
+const uint32_t kDefaultPortAllocatorFlags = 0;
+
```

Comment:
We don't use this flag and access the actual flags in a different way, but having this allows tests to be built and matches the upstream.

Reply:
I commented this out and left a comment here.